src/styles/card.scss

```diff
@@ -0,0 +1,115 @@
+@import "./variables";
```

Comment:
you can import variables in main.scss at the beginning as well

Reply:
Agree, import it in main.scss before other style files and variables will be availible in every file